frontend/src/helpers/index.ts

```diff
@@ -98,6 +119,9 @@ function getAppUrl(): string {
     exportFunctions.isDevMode() || exportFunctions.isElectron()
       ? 'http://localhost:4466'
       : window.location.origin;
+  if (exportFunctions.isDockerDesktop()) {
+    url = 'http://localhost:64446';
+  }
```

Comment:
I wonder if we should just set up an env var with the port and check it from here instead?